app/views/desktop/home.ejs

```diff
@@ -23,27 +23,24 @@ this license in a file with the distribution.
   <div id="main-container" class="small-12 medium-12 large-9 columns">
     <%- include('../_partials/flash') -%>
     <div id="main-feed" class="row">
-      <h4>1965: A Group starts to form</h4>
-      <p>Students at University of Illinois start to create a group for those interested in computing, ultimately leading to the formation of ACM@UIUC</p>
-      <h4>1967: ACM@UIUC gets its charter</h4>
-      <p>On Nov 1st, 1967 the Association for Computing Machinery Student Chapter at the University of Illinois was chartered to create an organization for those interested in Computers.</p>
-      <h4>1982: ACM@UIUC becomes 2nd largest ACM student chapter</h4>
-      <p>ACM@UIUC is the second largest ACM student chapter in the nation with 250 members. This year ACM's Team wins east section of the Central Regional Scholastic Programming Contest, a programming contest in FORTRAN.</p>
-      <h4>1993: Caffeine comes online</h4>
-      <p>A staple of the office, Caffeine was an internet connected vending machine, one of the first devices connected to internet connected, and was maintained by SIGArch and later SIGBot. It has undergone multiple upgrades and overhauls overtime including the current version called Merch.</p>
-      <h4>1995: Reflections|Projections </h4>
-      <p>The first Midwestern ACM Chapter Conference, later named Reflections|Projections was held, starting a 23 year tradition of ACM@UIUC hosting students, academics and members of industry for a conference on the past and the future.
-        Notable speakers over the years included Steve Wozniak, Aaron Swartz and Max Levchin. R|P also has included an AI Programming competition called Mechmania since its inception.</p>
-      <h4>2014: First ever HackIllinois</h4>
-      <p>In 2014 the first HackIllinois occurred where students from UIUC and across the nation came to Siebel Center to hack for 36 hours. Over the years HackIllinois has been regarded in the hackathon community as an incredibly fun and well-run event, and more recently its focus on introducing students to Open Source has made it unique in the world of hackathons.</p>
-      <h4>2017: ACM@UIUC turns 50 years old</h4>
-      <p>ACM@UIUC turns 50 years old, with about 500 members, 18 SIGs, 2 flagship events and continues to be an open community for those interested in computing.</p>
-      <p></p>
+        <h4>Welcome to ACM@UIUC</h4>
+        <p>The Association for Computing Machinery (ACM) has been around for over 50 years and is a student group consisting of a variety of people with diverse backgrounds who share a common interest in computers.</p>
+        <a href="/about/history"><button id="history" class="button">More About ACM@UIUC's History</button></a>
```

Comment:
Stylistic choice, but I'm not sure this button is necessary for the home page. Feels better to just have a link to it under `/about`.

Reply:
I think it gets hidden in the about page, but it doesnt deserve to be on the nav bar so i though this was a good compromise

Reply:
I also added buttons for the other sections to make it standout less